prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,445 @@
+    "Provider": "AWS",
+    "Description": "Checklist when onboarding new AWS Accounts to existing AWS Organization.",
+    "Requirements": [
+      {
+        "Id": "deploy-an-account-from-predefined-IaC-template",
+        "Name": "Deploy an account from predefined IaC template",
+        "Description": "Deploy an account from predefined IaC template",
+        "Attributes": [
+          {
+            "Section": "Predefine-IAM-Roles",
+            "Name": "Predefine IAM Roles"
+          },
+          {
+            "Section": "Enabled-security-services",
+            "Name": "Enabled security services"
+          },
+          {
+            "Section": "IDC-integration-SSO-configuration",
+            "Name": "IDC integratio and SSO configuration for AWS Account"
+          },
+          {
+            "Section": "Organizarion-invitation",
+            "Name": "Organizarion invitation"
+          },
+          {
+            "Section": "Billing-emergency-and-security-contacts",
+            "Name": "Billing, emergency, and security contacts"
+          },
+          {
+            "Section": "Root-user-distribution-email-MFA",
+            "Name": "Root user have distribute email + MFA"
+          },
+          {
+            "Section": "S3-Block-Public-Access",
+            "Name": "S3 Block Public Access"
+          },
+          {
+            "Section": "Disable-AMI-public-sharing",
+            "Name": "Disable AMI public sharing"
+          }
+        ],
+        "Checks": []
+      },
+      {
+        "Id": "SPCS",
+        "Name": "SPCS",
+        "Description": "SPCS in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "SPCS-Based",
+            "Name": "Apply existing SPCS based on OU placement"
+          },
+          {
+            "Section": "Custom-SPCS",
+            "Name": "Add custom SPCS if required"
+          }
+        ],
+        "Checks": [
+          "iam_no_root_access_key"
+        ]
+      },
+      {
+        "Id": "Logging",
+        "Name": "Logging",
+        "Description": "Logging in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "DNS-Resolver-Queries-SIEM",
+            "Name": "Send DNS resolvers queries to SIEM"
+          },
+          {
+            "Section": "VPC-Flow-Logs",
+            "Name": "Send VPC flow logs (only denys) to S3 bucket"
+          },
+          {
+            "Section": "S3-Logs-Critical-Buckets",
+            "Name": "Send S3 access logs for critical buckets to separate S3 bucket"
+          },
+          {
+            "Section": "ALB-CFD-To-SIEM",
+            "Name": "Establish ready-to-be-enabled pipelines to deliver ALB and CFD to SIEM to toggle in case of emergency and investigations"
+          }
+        ],
+        "Checks": [
+            "cloudtrail_logs_s3_bucket_access_logging_enabled",
+            "cloudtrail_s3_dataevents_write_enabled",
+            "cloudtrail_cloudwatch_logging_enabled"
```

Comment:
`cloudtrail_cloudwatch_logging_enabled` does not apply here and there is check for VPC Flow Logs.

Reply:
Done!